content/en/docs/overview.md

```diff
@@ -1,29 +1,29 @@
 type: docs
 weight: 1
 ---
 
 {{% alert title="Important" color="info" %}}
-This article as well as the rest of this documentation describes the latest status of Notary Project and refers to it as The Notary Project. For documentation of older versions and implementations, please refer to the [previous Notary documentation](https://github.com/notaryproject/notary/tree/master/docs).
+This article as well as the rest of this documentation describes the latest status of the Notary Project and refers to it as the Notary Project. For documentation of older versions and implementations, please refer to the [previous documentation](https://github.com/notaryproject/notary/tree/master/docs).
 {{% /alert %}}
 
 ## Project Status
 
-The Notary project is in early development and design documents should not be considered final.
+The Notary Project is in early development and design documents should not be considered final.
 Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend]({{< ref "/community" >}}) the weekly meetings for details on the roadmap.
 
-## Notary Overview
+## Notary Project Overview
 
-![Notary scenarios](/docs/notary-e2e-scenarios.svg)
+![Notary Project scenarios](/docs/notary-e2e-scenarios.svg)
 
-Notary provides for multiple signatures of an [OCI Artifact][oci-artifacts] (including container images) to be persisted in an [OCI conformant][oci-distribution-conformance] registry.
+The Notary Project provides for multiple signatures of an OCI Artifact (including container images) to be persisted in an [OCI distribution v1.1 compliant][oci-distribution-conformance] registry.
 Artifacts are signed (`notation sign`) with private keys, and validated with public keys (`notation verify`).
 To support user deployment flows, signing an OCI Artifact will not change the `@digest` or `artifact:tag` reference.
 To support content movement across multiple certification boundaries, artifacts and their signatures will be easily copied within and across [OCI conformant][oci-distribution-conformance] registries.
 
-To deliver on the Notary goals of cross registry movement of artifacts with their signatures, changes to several projects are anticipated, including [OCI distribution-spec][oci-distribution-spec], [CNCF Distribution][cncf-distribution], [OCI Artifacts][oci-artifacts], [ORAS][oras] with further consumption from projects including [containerd][containerd], [OPA][opa], [Gatekeeper][gatekeeper] and the [docker client][docker-client].
+To deliver on the Notary Project goals of cross registry movement of artifacts with their signatures, changes to several projects are anticipated, including [OCI distribution-spec][oci-distribution-spec], [CNCF Distribution][cncf-distribution], [OCI Artifacts][oci-artifacts], [ORAS][oras] with further consumption from projects including [containerd][containerd], [OPA][opa], [Gatekeeper][gatekeeper] and the [docker client][docker-client].
```

Comment:
```suggestion
To deliver on the Notary Project goals of cross registry movement of artifacts with their signatures, changes to several projects are anticipated, including [OCI distribution-spec][oci-distribution-spec], [CNCF Distribution][cncf-distribution], [ORAS][oras] with further consumption from projects including [containerd][containerd], [OPA][opa], [Gatekeeper][gatekeeper] and the [docker client][docker-client].
```

Reply:
OCI artifact has been archived, so please also remove the reference at the bottom of this file

```
[oci-artifacts]:                https://github.com/opencontainers/artifacts
[oci-artifact-manifest]:        https://github.com/opencontainers/artifacts/pull/29
```

Reply:
fixed